Physicist/Physicist/XML/Templates/ActorTemplates.xslt

```diff
@@ -41,7 +41,7 @@
   </xsl:template>
 
   <xsl:template match="physicist:Actors/system:Mario">
-    <Player class="Player">
+    <Player class="Player" rotateSound ="sabredance">
```

Comment:
sabredance needs to be loaded by default inside of 'MainGame.cs' in order for this to be valid as hidden. If not, then the user will forget to add a soundeffect reference to it in the map and it will break without any warning or explanation other than 'Key not found: sabredance'. Which won't be on their map file